docs/modules/configuring/con-configuring-client-connections.adoc

```diff
@@ -82,4 +84,80 @@ If set to `REQUESTED`, the client is requested to present a certificate. If pres
 If set to `NONE`, client authentication is disabled.
 
 NOTE: The client's identity, as established through TLS client authentication, is currently not relayed to the target cluster. 
-For more information, see the https://github.com/kroxylicious/kroxylicious/issues/1637[related issue].
+For more information, see the {github-issues}/issues/1637[related issue].
+
+You may restrict the TLS protocols by either specifying an allow list of TLS protocols to be enabled, or
+specify a deny list of those protocols that should be disallowed. If you do not specify which protocols are to be used,
+the virtual cluster will use the defaults provided by the platform.
+
+When the client connects, the client and the virtual cluster will negotiate the highest mutually acceptable TLS protocol.
+If the two have no protocols in common, the connection will fail.
+
+The names of the TLS protocols are defined by {java-17-specs}/security/standard-names.html#sslcontext-algorithms[Java specification].
+
+.Example - restricting the TLS protocols using an allow list
+
+[source,yaml]
+----
+virtualClusters:
+  demo:
+    tls:
+      # ...
+      protocols:
```

Comment:
@k-wall elsewhere in the docs we say of the `tls` property:

> Use an empty object (`{}`) to inherit trust from the underlying platform on which the cluster is running. This option is suitable if the target cluster is using a TLS certificate signed by a public CA.

That idiom won't work for the case where we want to use the platform's truststore, but are opinionated about protocols and ciphers. We should reword that part of the docs, assuming it's possible to use an empty object just for the `tls.trust` property.

Reply:
Good point.   I've reworked the section to give a better introduction and incorporated that feedback too.